tests_integration/spcs/test_registry.py

```diff
@@ -3,7 +3,7 @@
 
 @pytest.mark.integration
 def test_token(runner):
-    result = runner.invoke_with_connection_json(["spcs", "registry", "token"])
+    result = runner.invoke_with_connection_json(["spcs", "image-registry", "token"])
```

Comment:
Not related to this PR. We should also have integration test for list-images and list-tags. We could discuss about how to add it offline

Reply:
Never mind. You already add them in https://github.com/Snowflake-Labs/snowcli/pull/703